packages/peregrine/lib/talons/CartPage/ProductListing/wishlistFragment.ee.js

```diff
@@ -0,0 +1,24 @@
+import { gql } from '@apollo/client';
+
+export const WishlistFragment = gql`
+    fragment WishlistFragment on Wishlist {
+        id
+        name
+        items: items_v2 {
```

Comment:
Generally I would avoid renaming or aliasing responses from gql as it is just another layer of abstraction that could make it more difficult to debug. If something went wrong with `items` in the response we would have to know that it was aliased from `items_v2`. Obviously this is a pretty minor thing, and I won't ask you to change it, but I think it's just good to be aware.

In this case I might actually leave it as `items_v2` is...rough.

Reply:
Yeah dude you are right. I am not a fan of renaming usually but I felt `items_v2` is not readable in the code.

Reply:
After discussion in the other PR I think we should move the alias to the destructure so we don't run into a conflict with the existing `items` schema value.